packages/parser/lib/Tokens.re

```diff
@@ -99,13 +99,18 @@ let token_to_string =
   | INTERPOLATION(v) => String.concat(".", v)
   | WS => " "
   | DOT => "."
-  | COMMA => ","
+  | COMMA => "COMMA"
   | ASTERISK => "*"
   | FUNCTION(fn) => fn ++ "("
   | NTH_FUNCTION(fn) => fn ++ "("
   | URL(url) => url ++ "("
   | BAD_URL => "bad url"
-  | BAD_IDENT => "bad indent";
+  | BAD_IDENT => "bad indent"
+  | PERCENTAGE(p) =>
+    "percentage: " ++ string_of_float(p) ++ string_of_char('%')
+  | NUMBER_CSS(n) => "(NUMBER '" ++ string_of_float(n) ++ "')"
+  | DIMENSION_CSS((f, s)) => "dimension: " ++ string_of_float(f) ++ s
+  | AT_KEYWORD(at) => "@ " ++ at;
 
 let token_to_debug =
```

Comment:
I wonder if we could remove `token_to_debug` and instead maintain `token_to_string`